README.md

```diff
@@ -40,3 +36,13 @@ To be able to run the e2e tests, Firefox must be installed and you must download
 
 To run all tests, run `pytest`.
 To run tests in a particular folder, run `pytest tests` or `pytest tests_e2e`.
+
+### Running the app in a VM
+In the root directory of this repo, run `vagrant up`.
+
+### Running the app via docker-compose
+For first time setup, run the following commands:
+Dev: `docker-compose up --build`
+Prod: `docker-compose -f docker-compose.prod.yml up --build`
+
+In subsequent runs you can omit the `--build` flag.
```

Comment:
Actually I would say you should only omit the --build flag during first time setup.

`--build` forces a build of the image - otherwise it will use the existing image even if the dockerfile has changed. The first time you run docker-compose up, it will always build an image. In subsequent runs you should include --build.